easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-intel-fix-loop.patch

```diff
@@ -0,0 +1,26 @@
+# Bug in intel compiler
```

Comment:
actually, it seems like it's a bug in LLVM not conforming to the C++ standard, see https://software.intel.com/en-us/forums/intel-c-compiler/topic/640702